react/features/overlay/functions.js

```diff
@@ -1,5 +1,8 @@
 // @flow
 
+import { navigateRoot } from '../mobile/navigation/rootNavigationContainerRef';
```

Comment:
I think this will break web, so it kinda validates that we should decouple this from the overlays.

Reply:
Tested web and it works fine. We need to separate native overlays from the web ones, because they are rendered in the same place. In this case, native PageReloadOverlay will also become a screen, right?

Reply:
Not sure TBH. At this point the reload overlay is nothing more than a dialog. Once we extract the connection screen we can indeed look into reworking that overlay.